src/store.ts

```diff
@@ -49,19 +76,25 @@ export default new Vuex.Store({
     addCycle(state) {
       const newCycle: ICycle = {
         cursor: 1,
-        length: state.asm.iteration + 3,
+        length: state.asm.iteration + state.asm.activators + state.asm.nonActivators + 1,
```

Comment:
@navyhockey56 I'd add a new `getter` for `activators`, `nonActivators`, and maybe even a getter that would be `state.asm.iteration + state.asm.activators + state.asm.nonActivators + 1`, maybe you could call it `asmLength` or something like that.

Reply:
👍 sounds like a good idea to me